cli.js

```diff
@@ -96,7 +103,7 @@ api.run()
 		logger.exit(api.failCount > 0 || api.rejectionCount > 0 || api.exceptionCount > 0 ? 1 : 0);
 	})
 	.catch(function (err) {
-		if (err instanceof Error) {
+		if (err.name === 'Error') {
```

Comment:
Because we use `Error` for our internal messages (like the in the comment above), which don't need a stack and just informational. 

Errors that need stack trace are most of the time (I think) `TypeError`, `ParseError`, etc.